src/main/java/com/intelliChef/view/ViewManager.java

```diff
@@ -4,18 +4,21 @@
 import com.intelliChef.entities.Ingredient;
 import com.intelliChef.use_case.IngredientRepository;
 import com.intelliChef.use_case.analyzeImage.AnalyzeImageInteractor;
+import com.intelliChef.entities.Recipe;
```

Comment:
ViewManager shouldn't access entities. Create an interface for this. Look at the code for IngredientListRepository and how it implements an interface for access to ingredients.